package.json

```diff
@@ -33,6 +33,7 @@
     "help": "dt -h"
   },
   "dependencies": {
+    "dt": "0.0.1"
```

Comment:
ah, `dt` command is included in `definition-tester`.
Travis CI is passed in master/HEAD.
What version of npm do you use?